src/components/DonorDashboard/DonationDetails.jsx

```diff
@@ -0,0 +1,137 @@
+import React from 'react';
+import { Tag, TagLabel, Box, Text, Grid, GridItem, IconButton, Divider } from '@chakra-ui/react';
+import { DeleteIcon, EditIcon } from '@chakra-ui/icons';
+import { PropTypes } from 'prop-types';
+import DonationFurnitureContainer from '../InventoryPage/DonationFurnitureContainer';
+import DonationImagesContainer from '../InventoryPage/DonationImagesContainer';
+
+const DonationDetails = ({ data }) => {
+  // const {
+  //   id,
+  //   status,
+  //   firstName,
+  //   lastName,
+  //   submittedDate,
+  //   addressStreet,
+  //   addressUnit,
+  //   addressCity,
+  //   addressZip,
+  //   email,
+  //   phoneNum,
+  //   notes,
+  //   pictures,
+  //   furniture,
+  //   pickupDate,
+  //   routeId,
+  // } = data;
+  const formatDate = date => {
+    const d = new Date(date);
+    return d.toLocaleDateString('en-US', {
+      year: 'numeric',
+      month: 'numeric',
+      day: 'numeric',
+    });
+  };
+
+  // const [donation, setDonation] = useState();
+  const displayStatusTag = () => {
```

Comment:
Statuses were updated, so these no longer match. Should use enum instead of individual if statements. [updated statuses](https://github.com/ctc-uci/patriots-and-paws-frontend/blob/dev/src/utils/config.js#L7-L15)